app/workers/send_chase_email_to_candidates_worker.rb

```diff
@@ -1,3 +1,5 @@
+# TODO: Delete as part of continuous applications cleanup
```

Comment:
Will get removed here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/8867/files#diff-4e23a2a6ef5bad43b3153a150f701be0e53023530abf4c71dea89dff9f94db92